src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: block;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  box-sizing: border-box;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+    padding: 0;
+  }
+
+  &__content {
+    margin: 0 16px 16px;
+    padding: 0;
+    width: 166px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__description {
+    margin: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
```

Comment:
400 is default value for `font-weight` remove it everywhere according previous review